app/models/developer.rb

```diff
@@ -0,0 +1,18 @@
+class Developer < User
+  has_many :businesses
```

Comment:
I'd do this through an assignments table - so developer has_many assignments then has_many businesses through assignments. It's really a M-M rather than a 1-M relationship.